cdc_fuse_fs/constants.h

```diff
@@ -19,15 +19,22 @@
 
 namespace cdc_ft {
 
-// FUSE prints this to stdout when the binary timestamp and file size match the
-// file on the workstation.
-static constexpr char kFuseUpToDate[] = "cdc_fuse_fs is up-to-date";
+// FUSE prints
+//   Port 12345 cdc_fuse_fs is up-to-date
+// to stdout when its version matches the version (=build version or
+// size/timestamp for DEV builds) on the local device. The port is the gRPC port
+// that FUSE will try to connect to.
+static constexpr char kFusePortPrefix[] = "Port ";
```

Comment:
Nit: Can you use `std::string_view` here and elsewhere?

Reply:
Using std::string_view would introduce an extra (abeit cheap) indirection here, so I prefer using chars. Style guide seems to allow both: "Global strings: if you require a named global or static string constant, consider using a constexpr variable of string_view, character array, or character pointer, pointing to a string literal."